lib/rules/length-zero-no-unit/__tests__/index.js

```diff
@@ -258,6 +262,15 @@ testRule(rule, {
       line: 1,
       column: 27
     },
+    {
+      description: "ignore calc. has another zero units",
+      code: "a { padding: calc(1in + 0in * 2)) 0in calc(0px) 0px }",
+      fixed: "a { padding: calc(1in + 0in * 2)) 0 calc(0px) 0 }",
+
+      message: messages.rejected,
+      line: 1,
+      column: 36
```

Comment:
And other hardcore example: `padding: calc(var(--foo, 0in) + 10px)` :smile:

Reply:
`ignore: ["custom-properties"]` does not work with such example, is it a bug?

Reply:
i think yes, we catch a bug